cfpb/cfgov-refresh
# Fix background color and positioning of video play button

Fixes https://[GHE]/CFGOV/platform/issues/3525 (issues added in https://github.com/cfpb/cfgov-refresh/pull/5162)

Fixes disabled hover state on video play button by adding `!important` on hover state to override background-color `!important` that needed to be added when the play button is a link.

## Changes

- Fixes disabled hover state on video play button.
- Fixes positioning when the play button is in the lower-right.

## Testing

1. Pull branch, `gulp clean && gulp build`
1. visit http://localhost:8000/about-us/events/archive-past-events/cfpb-symposium-abusive-acts-or-practices/
1. Note that video play button changes on hover and there is margin to the right and bottom of the play button.
